portfolio/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Rachel Greenwood Portfolio</title>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="index.css" />
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=DotGothic16">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Silkscreen">
+    </head>
+    <body>
+        <nav><header><a href="#about">About</a><a href="#projects">Projects</a><a href="#contact">Contact</a></header><p id="now-playing">Now Playing: Selino - Tim Kahn</p><audio style="display:none" autoplay loop><source src="698343__timkahn__selino.wav"></audio></nav>
```

Comment:
I think your audio tag is malformed? https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide Happy to jam on this with you at our next sit down.